yarn-project/acir-simulator/src/public/executor.ts

```diff
@@ -33,9 +34,6 @@ import { CommitmentsDB, PublicContractsDB, PublicStateDB } from './db.js';
 import { PublicExecution, PublicExecutionResult } from './execution.js';
 import { ContractStorageActionsCollector } from './state_actions.js';
 
-// Copied from crate::abi at noir-contracts/src/contracts/noir-aztec/src/abi.nr
```

Comment:
I noticed this should come from constants gen when reading through so decided to update it